packages/wdio-cli/src/interface.js

```diff
@@ -116,10 +117,23 @@ export default class WDIOCLInterface extends EventEmitter {
         const runningJobs = this.jobs.size
         const isFinished = runningJobs === 0
 
+        if (this.sigintTriggered) {
+            clearTimeout(this.interval)
+            this.interface.log('\n')
+            return this.interface.log(!isFinished
+                ? 'Ending WebDriver sessions gracefully ...\n' +
+                '(press ctrl+c again to hard kill the runner)'
+                : 'Ended WebDriver sessions gracefully after a SIGINT signal was received!'
+            )
+        } else if(isFinished) {
```

Comment:
I don't think `this.sigintTriggered` is in any relation to `isFinished` and therefor there should be no else if and just a new if